invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/icon.overrides

```diff
@@ -28,3 +28,7 @@
     z-index: 1;
   }
 }
+
+.right-floated {
+  float: right;
```

Comment:
we already have this class defined in site.overrides, most likely it just misses `!important` clause if it didn't take effect on the icon